doc/TSCScope.md

```diff
@@ -0,0 +1,31 @@
+Subject to the contraints defined in the foundation
+charter, these are the areas that the TSC has autonomy
+in making decisions (some of which can be delegated to
+other groups, for example the CTC).  Outside of these
+areas the TSC will collaborate with the board
+when making decisions:
+
+* Code/Doc/ changes to any of the repositories under github/nodejs
```

Comment:
in that case, can we change it to read `Changes to the contents of any of the...` or something similar? This way it's clear that we have scope over everything in the repos, not just certain types of content.